src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -1,14 +1,6 @@
 """ :mod: RucioSynchronizer
 
   Agent that synchronizes Rucio and Dirac
-
-The following options can be set for the RucioSynchronizerAgent.
-
-.. literalinclude:: ../ConfigTemplate.cfg
-  :start-after: ##BEGIN RucioSynchronizerAgent
-  :end-before: ##END
-  :dedent: 2
-  :caption: RucioSynchronizerAgent options
```

Comment:
This should definitely stay. It builds the documentation readthedocs: https://dirac.readthedocs.io/en/latest/CodeDocumentation/ConfigurationSystem/Agent/RucioSynchronizerAgent.html#id1 (also see the other agent)